tutorials/Homology-Modelling/homology-modelling-tutorial.ipynb

```diff
@@ -177,21 +193,19 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "! cp nachr_alignment.pir alpha_pairwise_alignment.ali"
+    "! cp nAChR_model/nachr_alignment.pir nAChR_model/alpha_pairwise_alignment.ali"
    ]
   },
   {
+   "attachments": {},
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "- Although we copied the file and called it alpha_pairwise_alignment.ali it currently does not have the .ali format. First remove all subunits that you wont include in your homology model. \n",
-    "    \n",
-    "- A quick way of doing this is within VIM. Open your .ali file and hold 'shift :'. Then type\n",
+    "- Although we copied the file and called it alpha_pairwise_alignment.ali it currently does not have the .ali format. \n",
     "\n",
-    "    `g/Sequence_to_be_deleted/d`\n",
+    "- First remove all subunits that you will NOT include in the homology modelling process (i.e. everything except the alpha template (ACHA3_HUMAN) and our sequence of interest (ACHA_HUMAN)).  You can do this with any editor (vim, nano, notebook, nedit, gedit) \n",
```

Comment:
Agreed, this is much safer. The other machine independent option here would be to use `sed`, but that's also kinda complicated.